frontend/src/api/error-handler.ts

```diff
@@ -0,0 +1,23 @@
+import { AxiosError } from "axios";
+import { toast } from "react-toastify";
+
+export const errorHandler = (error: AxiosError) => {
```

Comment:
lets check if I understand it properly
1) first `error` is an instance of `AxiosError`
2) later `error` is resigned as something from  `error.response.data.error` ?

if steps 1 and 2 are true this is not good

Reply:
I will rename the variable and improve the code